app/src/main/java/ar/com/wolox/android/example/utils/StringUtils.java

```diff
@@ -0,0 +1,26 @@
+package ar.com.wolox.android.example.utils;
+
+import android.support.annotation.Nullable;
+
+/**
+ * Utilities to work with strings;
+ */
+public final class StringUtils {
+
+    static final String EMAIL_REGEX = "[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)" +
```

Comment:
Android has a way to check the emails. Use android.util.Patterns.EMAIL_ADDRESS.matcher(target).matches(); on the Fragment